docs/examples/advanced-cue/app1.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: core.oam.dev/v1alpha2
+kind: Application
+metadata:
+  name: testapp
+spec:
+  components:
+    - name: express-server
+      scopes:
+        healthscopes.core.oam.dev: testapp-default-health
+      settings:
+        cmd:
+          - node
+          - server.js
+        image: oamdev/testapp:v1
+        port: 8080
+      traits:
+        - name: new-route
+          properties:
+            domain: test.my.domain
+            http:
+              myservice: 8080
```

Comment:
`/myservice: 8080` 
Create Ingress failed. HTTPIngressPath.Path must begin with a '/'.

Reply:
yes, it's wrong, will fix it